web/mass_class/rtmp_bridge.py

```
@@ -0,0 +1,178 @@
+
+        if session_id in self.processes:
+            process = self.processes[session_id]
+            try:
+                process.terminate()
+                await process.wait()
+            except:
+                # Force kill if terminate fails
+                try:
+                    os.kill(process.pid, signal.SIGKILL)
+                except:
+                    pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)